code-challenges.test.js

```diff
@@ -55,3 +121,27 @@ var testArray2 = [7, 8, 2, 3, 1, 5, 4]
 
 
 // b) Create the function that makes the test pass.
+
+// const merger = (arr1,arr2) =>{
+//   let mergedArr = arr1.concat(arr2)
+//   let newArr = mergedArr.filter(function(value, index){
+//       return mergedArr.indexOf(value) == index
+//   }) 
+//   return newArr
+// }
+
+const merger = (arr1, ...arr) =>{
```

Comment:
this is good, but there is a away to use the spread operator here that doesn't require arr1 (because the spread operator includes it for us).